package.json

```diff
@@ -4,26 +4,29 @@
   "description": "An adapter for connecting Open MCT with YAMCS",
   "main": "dist/openmct-yamcs.js",
   "scripts": {
+    "clean": "rm -rf ./dist ./node_modules ./package-lock.json",
     "test": "echo \"Error: no test specified\" && exit 1",
     "lint": "eslint src example",
     "lint:fix": "eslint src example --fix",
     "build:dist": "NODE_ENV=production webpack",
-    "prepublish": "npm install openmct@unstable --no-save",
     "start": "NODE_ENV=dev webpack-dev-server",
     "prepare": "npm run build:dist"
   },
   "keywords": [
-    "openmct",
     "yamcs"
   ],
   "author": "National Aeronautics and Space Administration",
   "license": "Apache-2.0",
+  "dependencies": {
```

Comment:
This will cause Open MCT to be fetched and compiled whenever openmct-yamcs is included, which we don't want. Open MCT is a peer dependency of openmct-yamcs, not a direct dependency.